donkeycar/templates/complete.py

```diff
@@ -534,7 +534,7 @@ def run(self, mode, recording):
             cfg.SSD1306_ETH_INTERFACE_NAME,
             cfg.SSD1306_WLAN_INTERFACE_NAME
         )
-        V.add(oled_part, inputs=['recording', 'tub/num_records', 'user/mode'])
+        V.add(oled_part, inputs=['recording', 'tub/num_records', 'user/mode'], threaded=True)
```

Comment:
I am confused. The original `OLEDPart` was actually threaded, so did we go from a threaded part -> not threaded -> and not back to threaded again ?

Reply:
Yes, indeed. The design went from `threaded -> not thread -> thread` over the iterations. I benchmarked every iteration to get the best solution on this:

* original OLED part: threaded -> result: high display update rate, decent i2c actuarors, no graphics, noticable CPU usage on Raspberry Pi 3+
* first iteration: reworked not threaded OLED part -> result: poor display update rate, very responsive i2c actuators, graphics support, moderate CPU usage
* second iteration: reworked threaded OLED part -> result: decent update rate, very responsive i2c actuators, graphics support, moderate CPU usage